src/controllers/v1/entities.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * name : entity.js
+ * author : Aman Gupta
+ * created-date : 04-Nov-2021
+ * Description : Entity Controller.
+ */
+
+// Dependencies
+const entityService = require('@services/entities')
+
+module.exports = class Entity {
+	/**
+	 * create entity
+	 * @method
+	 * @name create
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - entities creation object.
+	 */
+
+	async create(req) {
+		const params = req.body
+		try {
+			const createdEntity = await entityService.create(params, req.decodedToken.id)
+			return createdEntity
+		} catch (error) {
+			return error
+		}
+	}
+
+	/**
+	 * updates entity
+	 * @method
+	 * @name update
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - entities updation response.
+	 */
+
+	async update(req) {
+		const params = req.body
```

Comment:
@adithyadinesh0412 we dont define it here, directly pass req.body and also instead of id pass req.params.id in function